docs/java-rest/high-level/usage.asciidoc

```diff
@@ -62,6 +62,37 @@ dependencies {
 }
 --------------------------------------------------
 
+[[java-rest-high-usage-maven-lucene]]
+==== Lucene Snapshot repository
+
+The very first releases of any major version (like a beta), might have been built on top of a Lucene Snapshot version.
+In such a case, you will need to define the Lucene Snapshot repository within your project.
```

Comment:
Maybe "In such a case you will be unable to resolve the Lucene dependencies of the client."